core-ui/public/i18n/en.yaml

```
@@ -426,7 +426,7 @@ common:
     error: Error
     generate-name: Generate name
     hide: Hide read-only fields
-    k8s-name-input: "Name must consist of lowercase alphanumeric characters, can contain '-' and '.' (e.g.: 'my.name-1')."
+    k8s-name-input: "Name must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name', or '123-abc')."
```

```suggestion
    k8s-name-input: "Name must consist of lowercase alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name', or '123-abc')."
```

I think it should stay as it was. [K8s docs](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names) use `lowercase` instead of `lower case`, but it will be verified by TWs.

---

Yes, let's keep `lowercase`.